exes/mqtt-broadcaster/src/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <async_mqtt/all.hpp>
+#include <boost/program_options.hpp>
+#include <tfc/ipc.hpp>
+#include "mqtt_broadcaster.hpp"
+
+auto main(int argc, char* argv[]) -> int {
+  try {
+    auto program_description{ tfc::base::default_description() };
+
+    std::string mqtt_host;
```

Comment:
I think it might be easier to edit the host via config interface, as this will be a common setting to edit

Reply:
moved this to config interface along with allowed_topics